lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -51,34 +51,40 @@ def world_transformation(xmin, xmax,
                      [0,    0,    0,    1]])
 
 
-def view_transformation(E, R, V):
+def rotation_about_vector(v, angle):
+    """
+    Produce a rotation matrix for an angle in radians about a vector.
+    """
+    v = v/np.linalg.norm(v)
```

Comment:
```suggestion
    vx, vy, vz = v / np.linalg.norm(v)
```
It's faster and more readable if you unpack the components instead of using all the index-access operations in the matrix below.

Reply:
Personally I find the indexing more intuitive to see the pattern, but don't feel strongly about it. Changed